libs/auth-api-lib/src/index.ts

```diff
@@ -68,6 +71,7 @@ export * from './lib/services/sequelizeConfig.service'
 export * from './lib/services/idp-provider.service'
 export * from './lib/services/translation.service'
 export * from './lib/services/delegations.service'
+export * from './lib/services/delegation-scope.service'
```

Comment:
Would be better to actually specify "WHAT" this library is exporting, instead of just export everything. It makes the abstraction more clear, and it'll be easier to use the API of the library.

e.g. `'./lib/services/sequelizeConfig.service'` this should not be exported and should not be used anywhere else than in this library.

Reply:
The API's import this for the SequalizeModule in their app.module class
SequelizeModule.forRootAsync({
      useClass: SequelizeConfigService,
    }),

Reply:
We will take a look at this during refactor week.